RecoDataProducts/src/classes_def.xml

```diff
@@ -341,6 +341,19 @@
  <class name="mu2e::CrvCoincidenceClusterCollection" />
  <class name="art::Wrapper<mu2e::CrvCoincidenceClusterCollection>" />
 
+<ioread sourceClass="mu2e::CrvDigi"
+        source="std::array<int16_t,8> _ADCs"
+        checksum="[3596926525]"
+        targetClass="mu2e::CrvDigi"
+        target="_ADCs, _NZS"
+        include="array, vector">
+<![CDATA[
+  _ADCs.resize(onfile._ADCs.size());
+  for(size_t i=0; i<onfile._ADCs.size(); ++i) _ADCs[i]=onfile._ADCs[i];
+  _NZS=false;
+]]>
+</ioread>
```

Comment:
This portion in the classes_def.xml allows us to read older digis with the updated CrvDigi class.